lib/philomena_web/controllers/api/json/image_controller.ex

```diff
@@ -27,4 +35,72 @@ defmodule PhilomenaWeb.Api.Json.ImageController do
         render(conn, "show.json", image: image, interactions: interactions)
     end
   end
+
+  def create(conn, %{"image" => image_params} = params) do
+    user = conn.assigns.current_user
+
+    params =
+      params
+      |> Map.put("scraper_url", Map.get(params, "url"))
+      |> Map.put("anonymous", Map.get(params, "anonymous", false))
+      |> Map.put("source_url", Map.get(params, "source_url", ""))
+      |> Map.put("description", Map.get(params, "description", ""))
+      |> Map.merge(image_params)
+
+    case user do
+      nil ->
+        conn
+        |> put_status(:unauthorized)
+        |> text("")
+
+      _ ->
+        attributes = %{
```

Comment:
Use `UserAttributionPlug` to get the attributes, and just override the fingerprint